internal/dataplane/kong_client_golden_test.go

```diff
@@ -259,7 +259,7 @@ func runKongClientGoldenTest(t *testing.T, tc kongClientGoldenTestCase) {
 	// Create the translator.
 	logger := zapr.NewLogger(zap.NewNop())
 	s := store.New(cacheStores, "kong", logger)
-	p, err := translator.NewTranslator(logger, s, "", tc.featureFlags, fakeSchemaServiceProvier{})
+	p, err := translator.NewTranslator(logger, s, "", tc.featureFlags, fakeSchemaServiceProvier{}, "cluster.local")
```

Comment:
Can we use the const `DefaultClusterDomain` here instead of hardcoding it? I know it's just test, but I don't see why not 🤔

Reply:
👍 That would cause an import cycle so I needed to extract manager's consts like so https://github.com/Kong/kubernetes-ingress-controller/commit/dabe77fed657ea0781bea0afa9f770d44b9e88b2

PTAL